client/matchmaking/matchmaking-disabled-card.tsx

```diff
@@ -152,6 +130,35 @@ export function ConnectedMatchmakingDisabledCard({
       ) : (
         <Headline6>Soon™</Headline6>
       )}
-    </Contents>
+    </DisabledCard>
+  )
+}
+
+export interface PartyDisabledCardProps extends DisabledCardProps {
```

Comment:
I see zero relation between these two components, so I'm not sure why you felt the need to extend this interface. Just because they have the same props under the same names doesn't mean they should have this kind of relation, just write `className` and `type` here as well (but honestly `type` should probably be `matchmakingType` in this context)